classes/admin.php

```diff
@@ -747,6 +747,12 @@ public function getPage($path)
 
                 $header = ['title' => $data['title']];
 
+                // Merge the default page headers into $header
+                $default_headers = Yaml::parse(
+                  $this->grav['config']->get('plugins.admin.frontmatter.default_headers'));
+
+                $header = (object) array_merge((array) $header, (array) $default_headers);
+
```

Comment:
Ok, i'm going to propose something a bit radical.  I can see what you are doing and why it's in admin, but frankly I think we can improve this even more by breaking out this functionality out into a standalone **admin-default-frontmatter** plugin.  

What I mean by this is this code above would simple be an event along the lines of `onAdminCreatePageFrontmatter`, where we pass the default headers.  Then you would move your config and simple logic above to a simple plugin that listens to this one event. It would then return the merged headers.

I think this is a better approach because it allows more powerful functionality.  You could enhance your plugin to be a list of setups where you have wildcard routes with different page frontmatter.  So some defaults of any pages in `/blog/*` and another set of default frontmatter for `/news/*` for example.  Or Another plugin entirely that does some cleanup or formatting of titles. 

Anyway, it wouldn't be tied and limited to the admin, and would give you the freedom to develop your plugin as you wished without having to go through the admin plugin itself.

What this means is the following:

* I'll accept the fix below for the modular body classes.  That's a bug really and a good catch. 
* I'll add the event to make it possible to create a plugin doing exactly what you are already doing.
* You create a new plugin (we can help if you like) with the params and logic you already have in this PR.

What do you think?